localization/gnss_fetch.py

```diff
@@ -0,0 +1,39 @@
+"""Fetch NMEA-0183 RMC sentences and save longtitudes, 
```

Comment:
Can we use existing module instead of creating a new one? Or we can revise existing one together.

Reply:
Surely we can. I will modify them (gnss and cam) later.